apps/todoapp/src/app/todos/components/user-info/user-info.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '../../services/auth.service';
+import { DiscordUser } from '../../types/discordUser.interface';
+
+@Component({
+  selector: 'todolist-user-info',
+  templateUrl: './user-info.component.html',
+  styleUrls: ['./user-info.component.scss'],
+})
+export class UserInfoComponent implements OnInit {
+  user!: DiscordUser;
+
+  constructor(private authService: AuthService) {}
+
+  ngOnInit() {
+    this.authService.checkAuthStatus();
```

Comment:
this should not be needed

Reply:
it was there before in previous component but yea you are right it is working without this line